src/libraries/System.Net.NameResolution/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -66,29 +64,31 @@ public void GetHostEntry_InvalidHost_LogsError()
 
         [ConditionalFact]
         [PlatformSpecific(~TestPlatforms.Windows)]  // Unreliable on Windows.
```

Comment:
Does this still hold? Locally I can't repro any problems with this test (now after 900834ed89229c5394e81d4316fdc20fd3daa78d)

This got added in https://github.com/dotnet/corefx/pull/42428

Just want to double-check is this test revealed a miss in product-code.

Reply:
I'm not sure. Could be use CI stability while back. Perhaps best would be do this separately e.g. put this in and do separate PR to enable this in ci and watch the outcome. If we see failures, we can try to reproduce locally or we can try to collect more info.